.github/workflows/ci.yml

```diff
@@ -115,3 +119,34 @@ jobs:
       
       - name: Run build
         run: npm run build
+
+  build-docker-image:
```

Comment:
we only want to run `build-docker-image` when a new commit is merged to `main`.

we shouldn't run on `pull_request` (line 7)

maybe this job should be in a separate yml file in order to do that?

Reply:
actually, I think this ci should be triggered when we release a version.
anyway, I just use this for testing building and pushing.
I will fix this problem after everything is going well

Reply:
ah you're right